.github/workflows/build.yaml

```diff
@@ -70,9 +70,6 @@ jobs:
         with:
           directory: manifests
 
-      - name: Build binary
-        run: CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' .
```

Comment:
AFAICT this means the code will not be built on PRs. I think is is a major problem :-(